Content.Server/Radio/EntitySystems/HeadsetSystem.cs

```diff
@@ -53,6 +61,7 @@ private void OnSpeak(EntityUid uid, WearingHeadsetComponent component, EntitySpo
             && keys.Channels.Contains(args.Channel.ID))
         {
             _radio.SendRadioMessage(uid, args.Message, args.Channel, component.Headset);
+            _audio.PlayPvs(_radioSound, uid); // RMC14
```

Comment:
Can't remember 100% if this is supposed to play for others near you or not but I don't think it does, would need to check but for some reason starting 2 clients at once in byond locally is broken for me